src/main/commands/commandManager.ts

```diff
@@ -280,32 +282,49 @@ export class CommandManager implements ExtensionComponent {
         }
     }
 
+    private async doReconnect() {
```

Comment:
Let's add a description for this function and also modify the description for the below `doDisconnect(question: boolean) `, so thay the user can easily understand the difference between these two functions.

In addition, since the UI no longer refers to `Xray` but to the `JFrog Platform`, let's also modify all the descriptions in the code accordingly. For example - 
`Disconnect from Xray server.` should be changed to `Disconnect from the configured JFrog Platform.`